setup.py

```diff
@@ -98,7 +98,7 @@ def txt_read(*paths):
         "matplotlib>=1.5",
         "cartopy[plotting]",
         "h5py",
-        "wxPython==3.0.2",
+        "wxPython>=3.0.2",
```

Comment:
The current situation of wxPython library is a bit unclear: https://groups.google.com/forum/#!topic/wxpython-users/RO2yGI9-fNE

The version 3.0.3 is only present on these snapshots builds: https://groups.google.com/forum/#!topic/wxpython-users/RO2yGI9-fNE , I have attempted to use them, and they do not work out of the box. Most likely we need to apply some changes for the new API. But I would not go looking for them until we understand where the WxPython project is going. I have thoughts about switching to PySide (now that Pyside2 is under way and part of the Qt Project).. but it would be a lot of re-coding!

The reason of ==3.0.2 and not <=3.0.2 is that is the only >3.x present on the WxPython project page: http://www.wxpython.org/download.php. 
Actually, I believe that it could be >= 3.0.0 and <=3.0.2 but, if I well remember, we have been able to freeze only 3.0.2.

@ghisvail : have you been able to run the application with wxPython release different that 3.0.2? Did you try to freeze it on Debian?